ci/unit/docker-setup.sh

```diff
@@ -1,55 +1,59 @@
 #!/usr/bin/env bash
 
- # This is intended to be run the plugin's root directory. `ci/unit/docker-test.sh`
+# This is intended to be run the plugin's root directory. `ci/unit/docker-test.sh`
 # Ensure you have Docker installed locally and set the ELASTIC_STACK_VERSION environment variable.
 set -e
 
- VERSION_URL="https://raw.githubusercontent.com/elastic/logstash/master/ci/logstash_releases.json"
+pull_docker_snapshot() {
```

Comment:
This seems quite similar to https://github.com/logstash-plugins/.ci/blob/e824dc9bb5db58d04d86773ae30ec90d4a84626b/docker-setup.sh#L7-L12 there is any way to share it?